_docs/geyser/setup.md

```diff
@@ -39,64 +60,31 @@ You can install Floodgate (but not Geyser) on the back-end servers to improve sk
+#### Setup:
+1. Make sure Velocity/BungeeCord is up-to-date. The proxy can be newer than the servers behind it (also called backend servers). If you're running the backend servers on an older version then the current {{ site.data.versions.java }}, you can use the plugin [ViaVersion](https://www.spigotmc.org/resources/19254/). It is recommended to have the plugin on all backend servers where necessary, instead to having it on the proxy. 
+2. Download a jar of Geyser from the [build server](https://ci.opencollab.dev/job/Geyser/job/master/) depending on what platform your server runs on. See the [FAQ](/geyser/faq/#which-plugin-version-of-geyser-do-i-need) if you're confused about which build to download. 
+3. Put the Geyser jar file in your plugins folder and start up the server.
+4. Port-forwarding:
+- If you are using a hosting provider, you will likely need to change your Bedrock port in the `config.yml` file. Information on your hosting provider might be available on the [Supported Hosting Providers](/geyser/supported-hosting-providers/) page. If your hosting provider is not listed there, and you are unable to port-forward the default 19132 port on UDP, try setting "clone-remote-port" to true and connecting with the java ip and port. Any port will work, as long as the protocol UDP is not blocked or already in use by the query feature/another mod. 
+- If you are not using a hosting provider, you will need to use port forwarding to allow people out of your LAN to connect. You want to port forward 19132 UDP, instructions to do that for most routers can be found [here](https://www.lifewire.com/how-to-port-forward-4163829). Note: Any port will work, as long as UDP traffic is not blocked. Java uses the TCP protocol, so you will need to open both UDP and TCP if you wish to use the same port.
+- To check whether it is working correctly, you can use the `geyser connectiontest [yourIP]:[yourPort]` command in your server console to check if the server is reachable from the outside.
+-  _Restart your server if you edited your config! Simply reloading will not work._
+5. Besides the `port` entry in the _Bedrock Section_ of Geyser's `config.yml`, you rarely need to change anything. A few examples:
+- `address: 0.0.0.0` The IP address that will listen for connections. There is no reason to change this unless you want to limit what IPs can connect to your server or if your hosting provider instructs you to do so.
+- `port: 19132` The port bedrock players will use to connect. If the option `clone-remote-port` is set to true, this port is ignored, and you'll have to use the same port on Bedrock as you would on Java to connect.
+6. The _remote_ section in the `config.yml` determines, which Java proxy the Bedrock players join. The proxy then configures, which server the players are sent to.
+- `address: auto` This means, that Geyser configures the proxy's ip, port and auth-type itself. Leaving this at auto is recommended.
```

Comment:
```suggestion
- `address: auto` Leaving this at auto is recommended, because it means that Geyser configures the proxy's IP, port and auth-type itself.
```
reworded, feel free to reject